### ISSUE TITLE
Importing Slider from kivy.uix.slider

### ISSUE THREAD

**Issue by socialtrouble** [kivy/kivy#3413]
When I'm importing Slider through the console a white window appears and then nothing happens (it should just import it, I don't know why there's a white box) and when I import it trough a program the window appears but is also whitem, python crashes and the Kernel dies. I don't have any other issues with Kivy, pygame, or any other module. 

I am using WinPython on Windows 7 as my main python install, and I have installed Pygame andCython successfully in it. I also reinstalled Kivy, Pygame and Cython trough pip and still everything works except the slider widget. I would love to post an error but the console just doesn't print it for that problem. I've searched all of the possible solutions and none of them helped. Installing kivy and every other module was just fine, and I realise that nobody has the same problem. Oh, and I installed both Windows Visual Studio C++ 2008 & 2010 (it was pointed out that some modules have problems installing when one of them is absent.)

To sum it up, 
I use WinPython on Windows 7.
Cython, Pygame & Kivy are both successfully installed & work perfectly, except the slider which crashes without an error in a console.

What to do? Is there a dependency for sliders? Sorry if it's kind of an abstract problem because of nonpresent errors but still does anybody know why that happens?
![problem](https://cloud.githubusercontent.com/assets/12828032/8080247/19e8ab78-0f6b-11e5-9da6-6cceb2900e8d.png)
I also have to mention that every other program that I've built with Kivy works fine if it doesn't have sliders.

**Comment by dessant** [comment]
Could you share that code snippet you're running?

**Comment by socialtrouble** [comment]
![ok](https://cloud.githubusercontent.com/assets/12828032/8090199/d3724806-0fac-11e5-904d-f5c93bb02ee2.png)


Here's the code without from kivy.uix.slider import Slider. It works fine for me until I import sliders. Importing is the problem here because it crashed even before it gets to the classes (it doesn't call them). Do you need .kv file?

**Comment by dessant** [comment]
Should we OCR that to run it? :) An image, come on, please post the code.

I suspect it is triggered by the interactive launcher, importing in a normal app seems to work.

**Comment by dessant** [comment]
I can confirm that importing slider while using the interactive launcher makes python crash.

**Comment by socialtrouble** [comment]
Sorry, was in a bit of a hurry and only had a pic of it :/ Will paste it as soon as possible.
I'll try using slider without the interactive launcher, but when I import kivy.uix.slider into a console before using the interactive launcher it still crashes (the white box that I mentioned)

**Comment by socialtrouble** [comment]
Thank you! Slider now works just fine without the interactive launcher and it loads as it should in the console :)

**Comment by dessant** [comment]
The interactive launcher has been deprecated in https://github.com/kivy/kivy/commit/3f83ed2a7b7d463d6b721b737b1adc0ec30ea7af.
